gallery_dl/extractor/instagram.py

```diff
@@ -133,11 +134,14 @@ def _extract_postpage(self, url):
             'fullname': media['owner']['full_name'],
             'post_id': media['id'],
             'post_shortcode': media['shortcode'],
+            'post_pageurl': url,
```

Comment:
Please change it to `url` instead of `post_pageurl` to be consistent with other extractors.

Reply:
`url` is, more often than not the URL to the file being downloaded (not always, sometimes it's, as you say, the URL to the post page). I think it should be called `post_url` if anything, but, given that it is possible to re-construct the post URL from `post_shortcode`, I'm not sure if this value is even necessary.

Reply:
For the name, I agree with post_url.

Yes, both post_url and tags can be extracted from the already available metadata; it's a matter of convenience: in my use case, I always save metadata in a json file and I have to extract information from them; having those fields already in a suitable json string or array is more straightforward, I don't have to parse them with string tools (and of course for tags we already have --write-tags, so it's nice to have them available for that option).

Reply:
@mikf Yes, that was the rationale I had not added it in the first place.